test/validation_test.rb

```diff
@@ -91,11 +91,6 @@ class Measured::Rails::ValidationTest < ActiveSupport::TestCase
     refute thing.valid?
   end
 
-  test "validation fails if only if the unit is set" do
```

Comment:
I can see why the validation was originally added, but this makes more sense to me. Let AR handle presence validation if it's added in the models, no need to duplicate behaviour. Do you think it would be worth it to write a test for this?